### DISCUSSION TITLE
Caprover One click install Apache Superset?

### DISCUSSION THREAD

**Discussion by hermesalvesbr** [apache/superset#22359] [Q&A / Help]
Just one docker image for start the Apache Superset, this is possible?  Today we need to clone a git repository and docker up a compose file. 

It would make it a lot easier if it was just a docker-compose.yml that binds the superset for use. It would be great to create an image for caprover, for example, and install in one click. But with several machines (dockers) complicates a bit.

https://github.com/caprover/one-click-apps/issues/802 

Is there any image, perhaps unofficial that unifies everything (dockers images) that apache superset needs for a quick deploy via docker-compose.yml?

**Comment by nktnet1** [comment]
Interested in this also, for other services like [Coolify](https://github.com/coollabsio/coolify) and [Dokploy](https://github.com/Dokploy/dokploy).

Will look to see if I can create one.

**Reply by nktnet1** [reply]
### Update 1
Found this:
- https://github.com/amancevice/docker-superset

### Update 2
Using the above, I was able to deploy it successfully to both Dokploy and Coolify - pull requests below:
- https://github.com/Dokploy/dokploy/pull/1158
- https://github.com/coollabsio/coolify/pull/4891

Although given that they're unofficial images, it's not certain they'll be merged. However, you can follow the instructions/videos I linked in the PR to get it set up.

As for Caprover, it should be possible to convert the `docker-compose.yaml` file to caprover syntax as per the documentations: 
- https://caprover.com/docs/docker-compose.html

The only challenge currently is with the volume mount needing to be configured manually. I've posted a feature request here:
- https://github.com/caprover/caprover/discussions/2245
